backend/internal/server/apis/user/user.go

```diff
@@ -96,8 +102,29 @@ func createUserAccount(c *fiber.Ctx) error {
 		return responder.BadRequest(c, err.Error())
 	}
 
-	// TODO: Implement user creation through the DATABASE here
+	// parse actual user object
+	u := &models.User{}
+	err = c.BodyParser(&u)
+	if err != nil {
+		log.WithErr(err).Error(err.Error())
+		return responder.BadRequest(c, err.Error())
+	}
+
+	// write to database
+	db := db.GetSession(c)
+	log.Debug("Creating user %s", user.Email)
+	u, result := db.CreateUser(u)
+	if result != nil {
+		log.WithErr(err).Error(result.Error())
+		return responder.BadRequest(c, result.Error())
```

Comment:
This isn't necessarily a bad request, right? I think at this point this would be an internal error.  Thoughts?

Reply:
Fair point. I'll make that change.

Reply:
This has been changed in dc5834b.